backend/server/static/api/meetingbook.yaml

```diff
@@ -109,6 +133,19 @@ paths:
           schema:
             $ref: "#/definitions/Booking"
 definitions:
+  RegistrationBody:
+    type: "object"
+    required:
+      - email
+      - password
+      - password_confirm
```

Comment:
discussed in chat. 

I'd propose to remove password_confirm from the api

Reply:
fixed